src/custom_models_action.py

```diff
@@ -83,16 +83,6 @@ def run(self):
     def _prerequisites(self):
         """Check prerequisites before execution."""
 
-        supported_events = ["push", "pull_request"]
```

Comment:
Do we want the action to run on every event? (.e.g comments, etc)?

Reply:
I don't see any reason to limit what the user wants to do. E.g. in [global-models-env, I'm running on workflow-dispatch event](https://github.com/datarobot/global-envs-models/blob/main/.github/workflows/global_models_us_prod.yaml#L4)

Reply:
To save execution resources and redundant access to DR, you might want to specify the relevant events that you want the action to run.
You may just add any event that you think it worth to run the action.

Reply:
But, since you limit it in the workflow itself, we may relax it on the action. I'll approve.

Reply:
I actually ended up reverting it before seeing your comment but I'll add it back again as I still think it is up to the user to decide which events they want to trigger on. Thanks